.github/tests/nested/deps2-values.yaml

```diff
@@ -0,0 +1,21 @@
+spire-server:
+  enabled: false
+
+spire-agent:
+  enabled: true
+  server:
+    address: spire-server.nested-deps
+  serviceAccount:
+    name: spire-agent-upstream
+  agentSocketPath: /run/spire/agent-sockets-upstream/spire-agent.sock
+  bundleConfigMap: spire-bundle-upstream
+  healthChecks:
+    port: 9981
+
+spiffe-csi-driver:
+  enabled: true
+  pluginName: upstream.csi.spiffe.io
+  agentSocketPath: /run/spire/agent-sockets-upstream/spire-agent.sock
+  healthChecks:
+    port: 9810
```

Comment:
Because the test is trying to reproduce what someone might actually try to deploy on their systems. IE, root server (somewhere else) and a fully functional nested spire server (upstream agent, nested-spire-server, downstream agent, + the rest of the plumbing needed to make it work). Currently you can't deploy the upstream agent with the nested-spire-server so its a separate deployment for that piece. Follow-on pr that allows that to be merged together.